tests/components/test_population.py

```diff
@@ -0,0 +1,188 @@
+import pandas as pd
+import pytest
+
+from vivarium_census_prl_synth_pop.components.population import Population
+from vivarium_census_prl_synth_pop.constants import paths
+
+fake_household_1 = pd.DataFrame(
+    {
+        "household_id": [1] * 5,
+        "date_of_birth": [
+            pd.Timestamp("1987-03-01 00:00:00"),
+            pd.Timestamp("2023-05-11 00:00:00"),
+            pd.Timestamp("2026-04-01 00:00:00"),
+            pd.Timestamp("2025-11-21 00:00:00"),
+            pd.Timestamp("1992-07-25 00:00:00"),
+        ],
+        "age": [39, 8, 5, 7, 35],
```

Comment:
Could we just define age _or_ DOB and then calculate the other one? These are not consistent. IIUC, date of birth only matters in that it is sometimes before and sometimes after the simulation start, so the specific days of the year aren't necessary here.

Reply:
I am not sure I understand what you are saying.  It does matter the day because someone could be born the same year as the simulation starts so we would need to know if it is before or after that day.  Age and date of birth were not used to calculate the other because that allows us to test "bad" data if we wanted to (and maybe should?).  Maybe this isn't the best practice here.

Reply:
For example, we could test that a simulant is born in the simulation but is also the oldest member in that household.  Not possible but we would expect them to become the new reference person.

Reply:
[DOB and age will always be consistent within the simulation](https://github.com/ihmeuw/vivarium_census_prl_synth_pop/blob/8c72892600114944b06b7a353befea1a258220a6/src/vivarium_census_prl_synth_pop/components/population.py#L138-L140), but since this is a unit test it is okay not to enforce that.

I actually think what would be clearer for testing this unit would be to only have two dates of birth: one before the simulation start, and one after. That is all the method cares about.

The more information you have here with all these different dates of birth, the less clear it is what is important. That's why stripping it down to the bare minimum would help me follow.

It _is_ possible that someone is born in the simulation and is also the oldest member in the household, if their parent leaves (via death, domestic migration, emigration).

Reply:
Oops I deleted part of my last comment.  Yes they could be the oldest member.  I meant to say we could test a simulant is born in the simulation and is the oldest member in their household that includes their biological mother.  

I will update to only have 2 dates of birth because that is a great idea.